packages/twenty-front/src/modules/spreadsheet-import/steps/components/MatchColumnsStep/components/UnmatchColumnBanner.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { IconChevronDown, IconInfoCircle } from 'twenty-ui';
+const StyledBanner = styled.div`
```

Comment:
Okay, so you don't need to duplicate the Banner code here :)

To answer your question, you need to add "className" as a prop of the Banner to be able to "Styled" it through styled-component. Just adding this props will allow you to use `styled(Banner)` syntax

Reply:
Ah, I am so dumb! haha 
Thanks :)